source/Government.cpp

```diff
@@ -103,6 +103,9 @@ void Government::Load(const DataNode &node)
 			displayName = name;
 	}
 
+	// No need to consider clearing it if it's empty.
+	bool clearedRaids = !raidFleets.isEmpty();
```

Comment:
I meant what I said.
```suggestion
	// If raid fleets already exist, clear all existing raid fleets if the "add" or "remove" keywords aren't used
	// and a "raid" key is encountered.
	bool clearRaids = !raidFleets.isEmpty();
```

Reply:
😔